packages/color-field/README.md

```diff
@@ -33,7 +33,7 @@ The color field consists of several key parts:
 - **Size variations**: Different size options to match your design requirements
 
 ```html
-<sp-color-field value="#ffff00"></sp-color-field>
+<sp-color-field value="#ffff00">Background color</sp-color-field>
```

Comment:
Color field uses `TextfieldBase` which now includes the mixin for a slotted label